migrationtools/databases.py

```diff
@@ -118,6 +118,43 @@ def _setup_tables(self, meta):
         self.registration = meta.tables['registrations_registration']
         self.user = meta.tables['auth_user']
 
+    def create_registration_data(
+            self, operator_id, msisdn_registrant, msisdn_device, id_type,
+            language, consent, edd=None, faccode=None,
+            any_id_no=None, passport_origin=None, mom_dob=None):
+        data = {
+            'operator_id': operator_id,
+            'msisdn_registrant': msisdn_registrant,
+            'msisdn_device': msisdn_device,  # this will be the same as registrant when a user registers themselves.
+            'id_type': id_type,
+            'language': language,
+            'consent': consent
+        }
+        if id_type == 'sa_id':
+            data['sa_id_no'] = any_id_no
+        elif id_type == 'passport':
+            data['passport_no'] = any_id_no
+            data['passport_origin'] = passport_origin
+
+        if mom_dob is not None:
+            data['mom_dob'] = mom_dob
+        if edd is not None:
+            data['edd'] = edd
+        if faccode is not None:
+            data['faccode'] = faccode
+        return data
+
+    def create_registration(self, registrant_id, reg_type, data, source, created_at, updated_at):
+        uid = str(uuid.uuid4())
```

Comment:
how are UUIDs in the current system generated, using `str()` or with `.hex`, the output differs slightly:
```python
>>> uuid4().hex
'bf7cbdd63072488faa1fda4ace270db8'
>>> str(uuid4())
'56cfe785-f81d-4509-bf11-6421b344f140'
```